Rakefile

```diff
@@ -3,6 +3,8 @@ require "yast/rake"
 # do nothing in osc:build as it can be builds exclusivelly on s390
 Rake::Task["osc:build"].clear_actions if `uname --machine` !~ /s390/
 
+Yast::Tasks.submit_to((ENV["YAST_SUBMIT"] || :factory).to_sym)
```

Comment:
well, it is basically default. So jenkins work fine also without this line as YAST_SUBMIT is by default respected if not overwritten by submit_to